File: SDKs/github.com/ghes-2.21/1.1.4/go/pkg/models/operations/pullsdeletereviewcomment.go
package operations

import (
"openapi/pkg/models/shared")

type PullsDeleteReviewCommentPathParams struct {
    CommentID int64 `pathParam:"style=simple,explode=false,name=comment_id"`
    Owner string `pathParam:"style=simple,explode=false,name=owner"`
    Repo string `pathParam:"style=simple,explode=false,name=repo"`
    
}

type PullsDeleteReviewCommentRequest struct {
    PathParams PullsDeleteReviewCommentPathParams 
    
}

type PullsDeleteReviewCommentResponse struct {
    ContentType string 
    StatusCode int64 
    BasicError *shared.BasicError 
    
}

File: SDKs/github.com/api.github.com/1.1.4/typescript/src/sdk/models/operations/pullsdeletereviewcomment.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";


export class PullsDeleteReviewCommentPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=comment_id" })
  commentId: number;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=owner" })
  owner: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=repo" })
  repo: string;
}


export class PullsDeleteReviewCommentRequest extends SpeakeasyBase {
  @Metadata()
  pathParams: PullsDeleteReviewCommentPathParams;
}


export class PullsDeleteReviewCommentResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;

  @Metadata()
  basicError?: shared.BasicError;
}
